### ISSUE TITLE
GridSearch: specify learning rate range in search_space.json

### ISSUE THREAD

**Issue by martsalz** [microsoft/nni#1582]
I would like to use the RandomSearch tuner for the hyperparameter search. Using RandomSearch I would like to find the learning rate in the range 0.0001 to 0.1. Which command do I use to specify the range for the `search_space.json`?

https://github.com/microsoft/nni/blob/v1.0/docs/en_US/Tutorial/SearchSpaceSpec.md
